app/assets/javascripts/discourse/app/components/topic-list/list.gjs

```diff
@@ -101,7 +101,8 @@ export default class TopicList extends Component {
     return applyValueTransformer(
```

Comment:
It's not needed as it mostly syntactic sugar, but we have `applyMutableValueTransformer`